### DISCUSSION TITLE
I can't use GitHub copilot. “ You can't proceed with your payment”！！！

### DISCUSSION THREAD

**Discussion by 921491015** [community/community#127853] [GitHub Education]
### Hi everyone,

First, I applied for GitHub education. The process was difficult, but I finally passed. But when I tried to subscribe to GitHub copilot, I found that it was not available. The reason was:
![image](https://github.com/community/community/assets/91826865/0307575f-11cf-459f-9adc-2a3867b19d13)
I don't know what happened, and I can't change my payment information at this time, there is no edit option.
I am just a student who wants to use this advanced code tool, and my selected country and school are not on the US government's sanctions list. I hope to get a reasonable explanation.

**Comment by HDX-GOD** [comment]
一样的问题，不知道帖主怎么解决，蹲个后续

**Reply by nathanhsuuu** [reply]
+1

**Comment by S1lverLavender** [comment]
插眼

**Comment by xftxyz2001** [comment]
怎么样了？

**Reply by 921491015** [reply]
后面好了，成功通过了，重复申请了两次。
